web/src/stream_create.js

```diff
@@ -290,7 +310,7 @@ function create_stream() {
                 // with i18n.  And likely we should be reporting the
                 // error text directly rather than turning it into
                 // "Error creating stream"?
-                stream_name_error.report_already_exists(stream_name);
+                stream_name_error.report_already_exists();
```

Comment:
I don't believe this parameter was being passed, as report_already_exists had no parameters.

Reply:
Indeed, the best way to handle this is to put this change in its own mini prep commit at the start of the branch with the explanation you had in this comment there in the commit message; I can then just merge that commit the next time I review this.

Reply:
Fixed it.

Reply:
Merged this one as 7eb34529e6fe11d483a74a91ca5fc2fdc5967914